hilti/include/rt/backtrace.h

```diff
@@ -14,15 +14,18 @@ class Backtrace {
 public:
     Backtrace();
 
-    const auto& backtrace() const { return _backtrace; }
+    std::vector<std::string> backtrace() const;
 
-    friend bool operator==(const Backtrace& a, const Backtrace& b) { return a._backtrace == b._backtrace; }
+    friend bool operator==(const Backtrace& a, const Backtrace& b);
     friend bool operator!=(const Backtrace& a, const Backtrace& b) { return ! (a == b); }
 
 private:
-    std::vector<std::string> _backtrace;
+    int _frames = -1;
+    void* _callstack[64];
```

Comment:
Let's remove the extra padding.
```suggestion
    void* _callstack[64];
    int _frames = -1;
```